exercises/phone-number/description.md

```diff
@@ -11,12 +11,14 @@ The first three digits of the local number represent the *exchange code*, follow
 The format is usually represented as
 
 ```text
-(NXX)-NXX-XXXX
+NXX NXX-XXXX
```

Comment:
We might show both that are often used... 

The tests are not going to differentiate in a way that errors, but will present it as a cleaned version.

Here we are showing what is "usually" represented, and I might think that 

```text
(NXX) NXX-XXXX
```

And

```text
(NXX)NXX-XXXX
```

are pretty evenly tied in the usual use.

I totally agree with using either with parenthesis or or dash to separate the area code from either or both sides of that, never both at the same time.

Reply:
We could also change "usually" to "sometimes" and not worry about what is commonly used where :smile: